FLEDGE.md

```diff
@@ -350,7 +365,7 @@ In the long term, we need a mechanism to ensure that the after-the-fact reportin
 
 #### 5.1 Seller Reporting on Render
 
-The seller's JavaScript (i.e. the same script, loaded from `decisionLogicUrl`, that provided the `scoreAd()` function) can also expose a `reportResult()` function:
+The seller's JavaScript (i.e. the same script, loaded from `decisionLogicUrl`, that provided the `scoreAd()` function) can also expose a `reportResult()` function, which is called for both top-level seller worklets and the component auction worklet with the winning bid, if applicable:
```

Comment:
I'm not following.  There's only one top-level seller worklet.  I've made it singular, and beefed up the comment a bit, but not sure about the rest.  Also pretty sure bid should be singular, since only one bid wins an auction.